slider/library.py

```diff
@@ -4,6 +4,8 @@
 import os
 import pathlib
 from time import sleep
+from sys import platform as _platform
+from re import sub as _sub
```

Comment:
why are we aliasing the imports, if we want to namespace these we can just do `import sys` and `import re` and use the qualified names later.